src/history.cpp

```diff
@@ -71,10 +72,10 @@ namespace mamba
             std::cmatch ematch;
             std::vector<std::string> pkg_specs;
             const char* text_iter = elems.c_str();
-            while (std::regex_search(text_iter, (const char*) elems.c_str() + elems.size(), ematch, elems_pat))
```

Comment:
This is not really necessary, cpp can figure it out having just `const char*` as an input.